.github/workflows/rust.yml

```
@@ -104,15 +104,20 @@ jobs:
+        with:
+          cache-directories: |
+            ~/.cargo/registry/src/**/librocksdb-sys-*
+
       - name: cargo build
         run: cargo build --verbose
 
       - name: cargo test
         run: cargo test --verbose
 
       - uses: Swatinem/rust-cache@v2
+        if: ${{ needs.release-please.outputs.release_created }}
```

actionlint (1.7.4)

property "release-please" is not defined in object type {formatting: {outputs: {}; result: string}}

(expression)